cypress/integration/console/devices/import.spec.js

```diff
@@ -79,5 +79,80 @@ describe('End device messaging', () => {
         cy.findByText(/ID already taken/).should('be.visible')
         cy.findByText(/an end device with/, /is already registered/).should('be.visible')
       })
+    cy.visit(`${Cypress.config('consoleRootPath')}/applications/${appId}/devices`)
+    cy.findByText('some-fail-id').should('not.exist')
+  })
+
+  it('succeeds setting lorawan_version, lorawan_phy_version and frequency_plan_id from fallback values', () => {
+    const devicesFile = 'freqId-version-phy-device.json'
+    const fallbackValues = {
+      lorawan_version: 'MAC V1.0',
+      frequency_plan_id: '863-870 MHz',
+    }
+    cy.findByLabelText('File format').selectOption('The Things Stack JSON')
+    cy.findByLabelText('File').attachFile(devicesFile)
+    cy.findByLabelText('Frequency plan').selectOption(fallbackValues.frequency_plan_id)
+    cy.findByLabelText('LoRaWAN version').selectOption(fallbackValues.lorawan_version)
+
+    cy.findByRole('button', { name: 'Import end devices' }).click()
+    cy.findByText('Operation finished').should('be.visible')
+    cy.findByText('3 of 3 (100.00% finished)').should('be.visible')
+    cy.findByTestId('notification')
+      .findByText('All end devices imported successfully')
+      .should('be.visible')
+    cy.findByRole('button', { name: 'Proceed to end device list' }).click()
+    cy.location('pathname').should(
+      'eq',
+      `${Cypress.config('consoleRootPath')}/applications/${appId}/devices`,
+    )
+    cy.findByTestId('error-notification').should('not.exist')
+    cy.visit(
+      `${Cypress.config(
+        'consoleRootPath',
+      )}/applications/${appId}/devices/fallback-test-device/general-settings`,
+    )
+
+    cy.findByText('Network layer', { selector: 'h3' })
+      .closest('[data-test-id="collapsible-section"]')
+      .within(() => {
+        cy.findByRole('button', { name: 'Expand' }).click()
+        cy.findByLabelText('Frequency plan').should('be.visible')
+        cy.findByText(/863-870 MHz/).should('be.visible')
+        cy.findByLabelText('LoRaWAN version').should('be.visible')
+        cy.findByText(fallbackValues.lorawan_version).should('be.visible')
+      })
+    cy.visit(
+      `${Cypress.config(
+        'consoleRootPath',
+      )}/applications/${appId}/devices/fallback-test-nice-id/general-settings`,
+    )
+    cy.findByText('Network layer', { selector: 'h3' })
+      .closest('[data-test-id="collapsible-section"]')
+      .within(() => {
+        cy.findByRole('button', { name: 'Expand' }).click()
+        cy.findByLabelText('Frequency plan').should('be.visible')
+        cy.findByText('Europe 863-870 MHz (SF12 for RX2)').should('not.exist')
```

Comment:
```suggestion
        cy.findByText('Europe 863-870 MHz (SF9 for RX2 - recommended)').should('be.visible')
```

It's better to assert on the exact value that is expected. Otherwise, you leave room for other unexpected values.